docs/config.rst

```diff
@@ -0,0 +1,122 @@
+Configuration Options
+=====================
+``schema.registry.url``
+ URL for schema registry
+
+ * Type: string
+ * Default:
+ * Importance: high
```

Comment:
All of these are listed as high priority, which seems unlikely. Can we assign better values? confluent-common sorts configs by: required fields first (any that don't have a default value), sorted by importance, then alphabetically.